ci/Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'etc'
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "base"
+
+  if Vagrant.has_plugin?('vagrant-env')
+    config.env.enable
+  end
+
+  machine_name = ENV['MACHINE_NAME'] || ("vmck-vagrant-" + `hostname`.strip)
```

Comment:
Do we really need to set the machine name? This was useful back when we ran vagrant tests on digitalocean, so we could identify which machine was created by which test, but at this point it's just noise.